### PR TITLE
deployment notes

### PR DESCRIPTION
**Pull request by SPlanzer**
Fixes: #
#197 Deployment notes

### Notes for Testing:

...

#### Source Code Documentation Tasks:
- [ ] README updated (where applicable)
- [ ] CHANGELOG updated

#### User Documentation Tasks:
- [ ] Confluence user guide updated (where applicable)

#### Testing Tasks:
- [ ] Added tests that fail without this change
- [ ] All tests are passing in development environment
- [ ] Reviewers assigned

#### Pull Request Management:
- [ ] Linked to sub-task
- [ ] Linked to the epic

### CODE REVIEW

**Review by strk — APPROVED** [review]
LGTM

**Comment by SPlanzer** [comment]
Oops, Left a fie with the more significant changes out of this PR. 
I have since pushed these

File: src/NZGBplugin/README.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+### Release Strategy
+When preparing for a release, the release source code must be pushed to a release branch that
+follows the naming convention `release-x.y`.
+
+### Deploying the NZGB plugin to LINZ Plugin Repository
+The [LINZ Plugin Repository](https://github.com/linz/qgis-plugin-repository) segregates development
+and production versions of plugins. This allows the controlled release of plugins in development to plugin testers and production code to general users.
+
+To start the automated release process, a release branch needs to be tag following the below conventions.
```
Comment (billgeo) [inline src/NZGBplugin/README.md:19]: do you mean 'tagged' here?

**Review by billgeo — APPROVED** [review]
A minor typo. But otherwise looks good to me.

**Pull request merged**